templates/manifest.json

```diff
@@ -254,5 +257,12 @@
                 "timeoutSecs": 3600
             }
         }
+    ],
+    "replacements": [
+        {
+            "fromKey": "name",
+            "from": "getting_started_python",
+            "to": "python-start"
+        }
```

Comment:
Maybe we could have this as `aliases` directly on the template objects? Then the CLI could just go through the templates and find one matching by ID or by alias.

Reply:
Then we could add the `name` of each template as an alias, so that we don't break backwards compatibility.

Reply:
Ha! I had a similar idea that the CLI should look at `id` when `name` isn't available (hence the same ids an names for all new templates), but this takes it even further and I like it. 👍